biscuit-auth/src/format/schema.proto

```diff
@@ -178,3 +181,39 @@ message ThirdPartyBlockContents {
   required bytes payload = 1;
   required ExternalSignature externalSignature = 2;
 }
+
+message AuthorizerSnapshot {
+  repeated string symbols = 1;
+  optional uint32 version = 2;
+  repeated SnapshotBlock blocks = 3;
+  required SnapshotBlock authorizerBlock = 4;
+  repeated GeneratedFacts generatedFacts = 5;
+  repeated Policy policies = 6;
+  repeated KeyMap publicKeyMap = 7;
```

Comment:
i think this is not needed anymore, now that `SnapshotBlock` contains an optional key.

What's needed however, is a map for interned public keys

Reply:
fixed in [3509d3e](https://github.com/biscuit-auth/biscuit-rust/pull/127/commits/3509d3e8d0f1853871d3956f73ff2587422a1968)